go/cmd/vtctldclient/command/onlineddl.go

```diff
@@ -239,6 +258,82 @@ func commandOnlineDDLRetry(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// commandOnlineDDLThrottle throttles one or multiple migrations.
+// As opposed to *most* OnlineDDL functions, this functionality does not end up calling a gRPC on tablets.
+// Instead, it updates Keyspace and SrvKeyspace entries, on which the tablets listen.
+func commandOnlineDDLThrottle(cmd *cobra.Command, args []string) error {
```

Comment:
The only difference b/w the two handlers appears to the `Ratio` and `ExpiresAt` vallues in the rule. We could  abstract the rest of the code into a helper, and call it from both Throttle/Unthrottle.

Reply:
Done. The two functions are consolidated in a single helper function.